setup_wizard/misc_operations.py

```diff
@@ -192,6 +196,18 @@ def execute(self, context):
             )
         return {'FINISHED'}
 
+    def __get_body_diffuse_uv0_node_name(self, texture_node_names: TextureNodeNames):
+        is_genshin_impact_diffuse_node_name = texture_node_names.BODY_DIFFUSE_UV0
+        is_stellartoon_diffuse_node_name = f'Body{texture_node_names.DIFFUSE_UV0_SUFFIX}' if texture_node_names.DIFFUSE_UV0_SUFFIX else None
+        is_shader_common_diffuse_node_name = texture_node_names.DIFFUSE  # Nya222/PGR
+
+        # StellarToon before common shaders is important!!!
+        body_diffuse_uv0_node_name = \
+            is_genshin_impact_diffuse_node_name or \
+            is_stellartoon_diffuse_node_name or \
+            is_shader_common_diffuse_node_name  # Keep on bottom as last check
+        return body_diffuse_uv0_node_name
```

Comment:
Should've mentioned why it was important to keep on bottom as last check.

This is because the `texture_node_names.DIFFUSE` is common across StellarToon AND other common diffuse node names. So we need to check StellarToon before the others otherwise it may attempt to provide the incorrect `body_diffuse_uv0_node_name`